easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.1-GCC-10.3.0.eb

```diff
@@ -26,6 +29,8 @@ dependencies = [
     ('PMIx', '3.2.3'),
 ]
 
+toolchainopts = {'debug': True}
```

Comment:
@Flamefire We don't always want to enable debug symbols, do we?

Reply:
My bad... Wasn't intentionally